tables/it-it-comp8.utb

```diff
@@ -22,13 +22,13 @@
 # -----------
 
 #  Copyright (C) 2011 by Caterina Avoledo <catery81@yahoo.it>
-#  Copyright (C) 2012,2016 by Simone Dal Maso <simone.dalmaso@juvox.it>
+#  Copyright (C) 2012,2016,2021 by Simone Dal Maso <simone.dalmaso@gmail.com>
```

Comment:
Shouldn't the address in AUTHORS also be updated?

Reply:
Ah, yes, sure good catch